content/en/internal_developer_portal/software_catalog/entity_model/ai-generated-systems.md

```diff
@@ -0,0 +1,68 @@
+---
+title: AI-generated Systems
+description: Discover recommended System entities powered by Bits AI to organize your Software Catalog.
+disable_toc: false
+further_reading:
+- link: "/internal_developer_portal/software_catalog/entity_model/entity_types"
+  tag: "Documentation"
+  text: "Entity Types"
+- link: "/internal_developer_portal/software_catalog/set_up/discover_entities"
+  tag: "Documentation"
+  text: "Discover Entities"
+---
+
+## Overview
+
+AI-generated Systems surface recommended System entities that you can add to your Software Catalog. Bits AI generates these recommendations using:
```

Comment:
I just noticed that in your screenshot, the UI says that it's "Bits AI" making the recommendations. I wonder if that would make more sense than saying that Systems recommend System entities, which, although probably true, might feel a little more ambiguous?